newEvaluation.html

```diff
@@ -379,10 +380,14 @@
 					</li>
 				</ul>
 			</fieldset>
-			<button type = "submit" name = "submit">Submit</button>
+			<button id = "submitButton" type = "submit" name = "submit">SUBMIT</button>
```

Comment:
Most definitely! Imagine this: you need to develop a site that works well when viewed on desktops, tablets and mobile devices. For desktops, you have enough screen real estate and can include enough spacing in your page, no probz. So you put a bunch of `<br>`s in there and it looks fantastic!

For tablets, less space available, your `<br>`s still force the layout breaks, a bit too much spacing for your liking, but hey, it's not _that bad_, is it? But then, you try to view the page on your brand new, shiny smartphone... Some of this stuff's full of content, then suddenly you're scrolling through half a page worth of empty space... _"What's happening here?"_, you ask... well, your `<br>`s are still stack in that markup, still forcing the layout engine to break to a new line, and even media queries don't help you at this point, without some ugly hacks in there to make everything play along...

Long story short? Yep, **always** pick `margin-bottom` instead of `<br>` tags, _unless_ you're absolutely certain that a `<br>` tag is what you need (in which case, you might _still_ want to check with one of the trainers to make sure your assumption is correct :P).